terraform/fleet/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "static" {
```

Comment:
can you reformat this file please, thanks :)

Reply:
sure reformated all with terraform fmt